src/app/modules/project/details/details.component.ts

```diff
@@ -313,24 +311,17 @@ export class DetailsComponent implements OnInit {
    * Method to display the edit project button based on the current user and the project user.
    * If the user either has the ProjectWrite scope or is the creator of the project
    */
-  private determineDisplayEditProjectButton(): void {
+  private determineDisplayEditAndDeleteProjectButton(): void {
     if (this.currentUser == null || this.project == null || this.project.user == null) {
       this.displayEditButton = false;
-      return;
-    }
-    this.displayEditButton = this.project.user.id === this.currentUser.id;
-  }
-
-  /**
-   * Method to display the delete project button based on the current user and the project user.
-   * If the user either has the ProjectWrite scope or is the creator of the project
-   */
-  private determineDisplayDeleteProjectButton(): void {
-    if (this.currentUser == null || this.project == null || this.project.user == null) {
       this.displayDeleteProjectButton = false;
       return;
     }
-    this.displayDeleteProjectButton = this.project.user.id === this.currentUser.id;
+    if (this.project.user.id === this.currentUser.id ||
+        this.authService.getCurrentBackendUser().role.name === 'Administrator') {
```

Comment:
I'm also missing something for the data officer here. I think it should be relatively easy to fix. 

I think by adding a method in the auth service that checks whether the backend user has the same institution as the project owner and has the InstitutionProjectWrite scope (again, this requires updating the enum of scopes)